src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -704,6 +714,37 @@ export class ResultsViewPageStore {
         }
     }, {});
 
+    readonly heatmapMolecularProfiles = remoteData<MolecularProfile[]>({
+        await: ()=>[
+            this.molecularProfilesInStudies
+        ],
+        invoke:()=>{
+            const MRNA_EXPRESSION = "MRNA_EXPRESSION";
+            const PROTEIN_LEVEL = "PROTEIN_LEVEL";
+            const selectedMolecularProfileIds = stringListToSet(this.selectedMolecularProfileIds);
+
+            return Promise.resolve(_.sortBy(_.filter(this.molecularProfilesInStudies.result!, profile=>{
+                return (profile.molecularAlterationType === MRNA_EXPRESSION ||
```

Comment:
lets try to use map for these.

Reply:
?

Reply:
I mean the constants like MRNA_EXPRESSION should come from one place in our app

Reply:
oh yeah. do they?

Reply:
good point